fixtures/small/nested_destructuring_actual.rb

```diff
@@ -1 +1 @@
-_1, (_2, _3) = []
+_a, (_b, _c) = []
```

Comment:
Related to https://github.com/ruby/ruby/pull/3163 -- in Ruby 3, assigning to numbered param is now a syntax error. This _is_ technically a backwards-incompatible change for rubyfmt, but IMO I think that's fine.